src/frameworks/hibernate.md

```diff
@@ -0,0 +1,96 @@
+# Hibernate
+
+[Hibernate ORM](https://hibernate.org/) is an object-relational mapping tool for the Java programming language. It provides a framework for mapping an object-oriented domain model to a relational database. Hibernate handles object-relational impedance mismatch problems by replacing direct, persistent database accesses with high-level object handling functions.
+
+## Services
+
+### MySQL, MariaDB and PostgreSQL
+
+[MySQL, MariaDB](configuration/services/mysql.md), and [PostgreSQL](configuration/services/postgresql.md) are open-source relational database technologies.
+
+The first step is to choose the database that you would like to use in your project. Define the driver for [MySQL](https://mvnrepository.com/artifact/mysql/mysql-connector-java), [MariaDB](https://mvnrepository.com/artifact/org.mariadb.jdbc/mariadb-java-client) or [PostgreSQL](https://mvnrepository.com/artifact/postgresql/postgresql), and the Java dependencies. Then determine the SessionFactory client programmatically:
+
+#### MySQL
```

Comment:
We've managed to avoid a 4th level header so far, I think.  Let's keep it that way.

Reply:
Probably just break these 3 out to their own sections and eliminate the common header.